tests/push/test_productid_updates_versions.py

```diff
@@ -79,12 +68,6 @@ def test_updates_product_versions(monkeypatch, tmpdir):
     ctrl.insert_repository(repo7)
     ctrl.insert_repository(repo8)
 
-    # Set up cert parser to use a fake cert object (saves us having to explicitly
-    # generate certs with certain values)
-    monkeypatch.setattr(
-        rhsm.certificate, "create_from_file", lambda _: FakeCert(["a", "d"])
-    )
-
     # make a fake productid.
     # content doesn't matter since we patched the cert parser, it just has
```

Comment:
Minor: this block could also have been deleted.  The comment is now false and the block is creating an empty file which I believe is never accessed.

Reply:
Looks like it's still needed as `pubtools.pulplib._impl.fake.client.FakeClient._do_upload_file()` [opens](https://github.com/release-engineering/pubtools-pulplib/blob/master/pubtools/pulplib/_impl/fake/client.py#L386) that file for reading and fails if it doesn't exists.

Reply:
I see.  I was right about the comment being false at least (there is no patching of a cert parser).  I leave it to you to decide how much to care about the comment's accuracy.